Maestro/src/Microsoft.DotNet.Maestro.WebApi/Services/RequestValidator.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography;
+using System.Text;
+using System.Web.Http;
+
+namespace Microsoft.DotNet.Maestro.WebApi.Services
+{
+    public class RequestValidator
+    {
+        private HttpRequestMessage request;
+        private byte[] requestContent;
+        private const string SignatureHeaderName = "X-Hub-Signature";
+
+        public RequestValidator(HttpRequestMessage request, byte[] requestContent)
```

Comment:
To me the functionality feels like a static method, no instances required; but I don't know if you guys have an anti-static design principle.